File: Module 3/3A Javascript/JavaScript Coding Challenge/Task4.js
// // Print a table containing multiplication tables

for (let firstNumber = 1; firstNumber <= 100; firstNumber++){
    for (let secondNumber = 1; secondNumber <= 12; secondNumber++){
        const multiply = firstNumber * secondNumber;
        
        console.log(firstNumber + " * " + secondNumber + " = " + (multiply));
    
     }
 }


//const size = 10;
//for (let i = 1; i <= size; i++) {
    //let row = '';
    //for (let j = 1; j <= size; j++) {
    //    row += (i * j).toString().padStart(4, ' ');
    //}
    //console.log(row);
//}

File: Module 4/4C_Everyday_Go_To_Python/Python_Coding_Challenge.py/Task4.py
"""
Print a table containing multiplication tables.
"""
# Define the size of the multiplication table
table_size = 10

# Print the header row
print("   ", end="")
for i in range(1, table_size + 1):
    print(f"{i:4}", end="")  # Align numbers in columns
print()

# Print the table rows
for i in range(1, table_size + 1):
    print(f"{i:2} |", end="")  # Row header with a separator
    for j in range(1, table_size + 1):
        print(f"{i * j:4}", end="")
    print()  # New line after each row
